server/scripts/populate.py

```diff
@@ -1,138 +1,181 @@
+# Assumptions: each club has its own events. Many-to-many not supported
+# yet. See example clubs.yml file for example input. db.ini file must be
+# in same directory
+
 from configparser import ConfigParser
 from datetime import datetime
 from os import path
+from enum import Enum
+from marshmallow import Schema, fields, validate
```

Comment:
super nit: I think this import might be in the wrong place

Reply:
Not sure what you mean...things in the wrong order?

Reply:
yeah, I think it should be after pprint/uuid

Reply:
Like so?

`from configparser import ConfigParser
from datetime import datetime
from os import path
from enum import Enum

import uuid
import psycopg2
import yaml

from marshmallow import Schema, fields, validate`